frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { route } from '../../../utils/route';
+import { Coin, IBalance } from '../../../api/account';
+import { Button } from '../../../components/forms';
+import { Balances } from '../summary/accountssummary';
+import styles from './buyReceiveCTA.module.css';
+
+type TBuyReceiveCTAProps = {
+  balanceList?: [string, IBalance][];
+  code?: string;
+  unit?: string;
+};
+
+export const BuyReceiveCTA = ({ code, unit, balanceList }: TBuyReceiveCTAProps) => {
+  const { t } = useTranslation();
+  const onBuyCTA = () => route(code ? `/buy/info/${code}` : '/buy/info');
+  const onReceiveCTA = () => {
+    if (balanceList) {
+      if (balanceList.length > 1) {
+        route('accounts/select-receive');
+        return;
+      }
+      route(`/account/${balanceList[0][0]}/receive`);
+    }
+  };
+
+  return (
+    <div className={`${styles.main} columns-container`}>
```

Comment:
I can't find `columns-container` class, maybe `styles.main` is enough?

Reply:
Yes, seems  like it's an old CSS. styles.main is enough.